pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -4005,6 +4005,50 @@ func Test_server_Update(t *testing.T) {
 		}
 		return nil
 	}
+
+	/*
+		Update test cases (only test float32 unless otherwise specified):
+		- Equivalence Class Testing ( 1000 vectors inserted before a update )
+			- case 1.1: success update vector
+			- case 2.1: fail update with non-existent ID
+			- case 3.1: fail update with different dimension vector (type: uint8)
+			- case 3.2: fail update with different dimension vector (type: float32)
+		- Boundary Value Testing ( 1000 vectors inserted before a update )
+			- case 1.1: fail update with ""
+			- case 2.1: success update with ^@
+			- case 2.2: success update with ^I
+			- case 2.3: success update with ^J
+			- case 2.4: success update with ^M
+			- case 2.5: success update with ^[
+			- case 2.6: success update with ^?
```

Comment:
the value is set as Id?

Reply:
Yes. These values are ID. I will make it explicit too.